### ISSUE TITLE
models pull fails with CLI 1.17.3.post

### ISSUE THREAD

**Issue by ehsanmok** [polyaxon/polyaxon#1504]
As of last night the `plx models pull` is failing with 

```
Untarring the contents of the file ...00:13
Traceback (most recent call last):00:13
  File "/home/semaphore/.pyenv/3.8/bin/polyaxon", line 8, in <module>00:13
 00:13
  File "/home/semaphore/.pyenv/3.8/lib/python3.8/site-packages/click/core.py", line 1130, in __call__00:13
    return self.main(*args, **kwargs)00:13
  File "/home/semaphore/.pyenv/3.8/lib/python3.8/site-packages/click/core.py", line 1055, in main00:13
    rv = self.invoke(ctx)00:13
  File "/home/semaphore/.pyenv/3.8/lib/python3.8/site-packages/click/core.py", line 1657, in invoke00:13
    return _process_result(sub_ctx.command.invoke(sub_ctx))00:13
  File "/home/semaphore/.pyenv/3.8/lib/python3.8/site-packages/click/core.py", line 1657, in invoke00:13
    return _process_result(sub_ctx.command.invoke(sub_ctx))00:13
  File "/home/semaphore/.pyenv/3.8/lib/python3.8/site-packages/click/core.py", line 1404, in invoke00:13
    return ctx.invoke(self.callback, **ctx.params)00:13
  File "/home/semaphore/.pyenv/3.8/lib/python3.8/site-packages/click/core.py", line 760, in invoke00:13
    return __callback(*args, **kwargs)00:13
  File "/home/semaphore/.pyenv/3.8/lib/python3.8/site-packages/click/decorators.py", line 26, in new_func00:13
    return f(get_current_context(), *args, **kwargs)00:13
  File "/home/semaphore/.pyenv/3.8/lib/python3.8/site-packages/polyaxon/logger.py", line 75, in clean_outputs_wrapper00:13
    raise e00:13
  File "/home/semaphore/.pyenv/3.8/lib/python3.8/site-packages/polyaxon/logger.py", line 71, in clean_outputs_wrapper00:13
    return fn(*args, **kwargs)00:13
  File "/home/semaphore/.pyenv/3.8/lib/python3.8/site-packages/polyaxon/cli/models.py", line 509, in pull00:13
    pull_one_or_many_project_versions(00:13
  File "/home/semaphore/.pyenv/3.8/lib/python3.8/site-packages/polyaxon/cli/project_versions.py", line 492, in pull_one_or_many_project_versions00:13
    _pull(version.name)00:13
  File "/home/semaphore/.pyenv/3.8/lib/python3.8/site-packages/polyaxon/cli/project_versions.py", line 470, in _pull00:13
    pull_project_version(00:13
  File "/home/semaphore/.pyenv/3.8/lib/python3.8/site-packages/polyaxon/cli/project_versions.py", line 434, in pull_project_version00:13
    polyaxon_client.pull_version(00:13
  File "/home/semaphore/.pyenv/3.8/lib/python3.8/site-packages/polyaxon/client/decorators/client_handler.py", line 95, in wrapper00:13
    return f(*args, **kwargs)00:13
  File "/home/semaphore/.pyenv/3.8/lib/python3.8/site-packages/polyaxon/client/project.py", line 1413, in pull_version00:13
    self.download_artifacts_for_version(config=config, path=path)00:13
  File "/home/semaphore/.pyenv/3.8/lib/python3.8/site-packages/polyaxon/client/decorators/client_handler.py", line 95, in wrapper00:13
    return f(*args, **kwargs)00:13
  File "/home/semaphore/.pyenv/3.8/lib/python3.8/site-packages/polyaxon/client/project.py", line 1384, in download_artifacts_for_version00:13
    run_client.download_artifact_for_lineage(00:13
  File "/home/semaphore/.pyenv/3.8/lib/python3.8/site-packages/polyaxon/client/decorators/client_handler.py", line 95, in wrapper00:13
    return f(*args, **kwargs)00:13
  File "/home/semaphore/.pyenv/3.8/lib/python3.8/site-packages/polyaxon/client/run.py", line 1107, in download_artifact_for_lineage00:13
    return self.download_artifacts(00:13
  File "/home/semaphore/.pyenv/3.8/lib/python3.8/site-packages/polyaxon/client/decorators/client_handler.py", line 95, in wrapper00:13
    return f(*args, **kwargs)00:13
  File "/home/semaphore/.pyenv/3.8/lib/python3.8/site-packages/polyaxon/client/run.py", line 1193, in download_artifacts00:13
    return self.store.download_file(00:13
  File "/home/semaphore/.pyenv/3.8/lib/python3.8/site-packages/polyaxon/stores/polyaxon_store.py", line 267, in download_file00:13
    self.download(filename=local_path, params=params, url=url, **kwargs)00:13
  File "/home/semaphore/.pyenv/3.8/lib/python3.8/site-packages/polyaxon/stores/polyaxon_store.py", line 223, in download00:13
    filename = untar_file(00:13
  File "/home/semaphore/.pyenv/3.8/lib/python3.8/site-packages/polyaxon/utils/path_utils.py", line 206, in untar_file00:13
    with tarfile.open(filename) as tar:00:13
  File "/usr/lib/python3.8/tarfile.py", line 1608, in open00:13
    raise ReadError("file could not be opened successfully")00:13
tarfile.ReadError: file could not be opened successfully
```

**Comment by polyaxon-team** [comment]
This turned out to be a bug in the client, the faulty logic:
```python
has_tar = 'tar' in self._get_header_value(
    headers=response.headers,
    key="content-disposition",
)
```

**Comment by ehsanmok** [comment]
Was fixed in 1.17.3.post2
